example-apps/service-external-origin-env/frontend-service/Dockerfile.prod

```diff
@@ -0,0 +1,15 @@
+FROM node:8.5.0 AS build
+
+RUN mkdir /workspace
+WORKDIR /workspace
+
+COPY package.json .
+COPY yarn.lock .
+RUN yarn
+
+COPY . .
+RUN node_modules/.bin/webpack
+
+FROM nginx AS prod
+
+COPY --from=build /workspace/dist /usr/share/nginx/html
```

Comment:
Are we sure this Dockerfile works?

Reply:
Yes, I tested it. The only thing not hooked up is the env variable